performance_regression_operator_multi_tenant_test.py

```diff
@@ -38,35 +33,32 @@ def run_fstrim_on_all_db_nodes(self):
 
     def wait_no_compactions_running(self, *args, **kwargs):
         for tenant in self.tenants:
-            self.log.info("Waiting for compactions to finish on the '%s' DB cluster",
-                          tenant.db_cluster.name)
+            self.log.info("Waiting for compactions to finish on the '%s' DB cluster", tenant.db_cluster.name)
             tenant.wait_no_compactions_running(*args, **kwargs)
 
     def preload_data(self, compaction_strategy=None):
         def _preload_data(tenant):
             prepare_write_cmd = tenant.params.get('prepare_write_cmd')
             db_cluster_name = tenant.db_cluster.name
             if not prepare_write_cmd:
-                self.log.warning(
-                    "No prepare command defined in YAML for the '%s' cluster",
-                    db_cluster_name)
+                self.log.warning("No prepare command defined in YAML for the '%s' cluster", db_cluster_name)
                 return
             self.log.info("Running preload command for the '%s' cluster", db_cluster_name)
-            tenant.create_test_stats(
-                sub_type='write-prepare', doc_id_with_timestamp=True)
-            stress_queue, params = [], {
-                'prefix': 'preload-',
-            }
+            tenant.create_test_stats(sub_type='write-prepare', doc_id_with_timestamp=True)
+            stress_queue, params = (
+                [],
+                {
+                    'prefix': 'preload-',
+                },
+            )
```

Comment:
This change looks odd.
Better to make either one line with still both options on it or 2 lines - one per option.
But not `this`.